libsolidity/codegen/Compiler.h

```diff
@@ -35,7 +35,9 @@ class Compiler
 public:
 	explicit Compiler(bool _optimize = false, unsigned _runs = 200):
 		m_optimize(_optimize),
-		m_optimizeRuns(_runs)
+		m_optimizeRuns(_runs),
+		m_context(CompilationMode::Creation, &m_runtimeContext),
```

Comment:
Not sure it is a good idea to take a reference of a not yet initialized variable.